lib/myarray.rb

```diff
@@ -0,0 +1,22 @@
+class My_array
+
+
+
+ def my_inject array, &my_block
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `my_block`. If it's necessary, use `_` or `_my_block` as an argument name to indicate that it won't be used.